astroquery/mast/observations.py

```diff
@@ -7,10 +7,12 @@
 """
 
 
+import ctypes
```

Comment:
is this import used?

Reply:
No, this is a residual import from some troubleshooting I was doing, thanks for the catch.